brigade/templates/resources_videos.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+{% block title %}Videos{% endblock %}
+{% block description %}Recordings from events within the Brigade Network{% endblock %}
+{% block page_id %}videos{% endblock %}
+
+{# Conditionals based on what type of page is rendering #}
+{% if topic %}
+{% else %}
+{% endif %}
+
+{% block content %}
+<header id="intro" class="page-header slab slab-dark-blue">
+  <div class="grid-box">
+    <h1 class="title">Videos</h1>
+    <div class="subtitle">Recordings from events within the Brigade Network</div>
+  </div>
+</header>
+
+{% if not topic %}
+<section class="slab slab--compact">
+  <div class="grid-box">
+    <div class="width-one-half">
+      <div class="alert">
+        <div class="alert__icon">
+          <i class="fas fa-info-circle"></i>
+        </div>
+        <div class="alert__content">
+          Check out <a href="https://www.youtube.com/user/CodeforAmerica/videos" target="_blank">Code for America's YouTube</a> for more videos.
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+{% endif %}
+
+<section class="slab slab--compact">
+  <div class="grid-box">
+    {% if topic %}
+    <p><a href="{{ url_for('.resources_videos') }}">&larr; Back to all videos</a></p>
+    {% endif %}
+
+    <h2 class="slab__title">
+      {% if topic %}
+      Latest Videos Tagged "{{ topic }}"
+      {% else %}
+      Latest Videos
+      {% endif %}
+    </h2>
+  </div>
+
+  <div class="grid-box">
+    <div class="width-one-half">
```

Comment:
Nit: We can probably remove `<div class="width-one-half">` so that the description displayed on a single line.
e.g.:
![image](https://user-images.githubusercontent.com/8628090/40394278-a3ed372c-5dd7-11e8-8ce0-a24c55b861ae.png)

Reply:
Okay, I'm into it.